src/D2L.CodeStyle.TestAnalyzers/NUnit/CategoryAnalyzer.cs

```diff
@@ -268,15 +291,19 @@ AnalyzerOptions options
 				StringComparer.Ordinal
 			);
 
-			AdditionalText bannedListFile = options.AdditionalFiles.FirstOrDefault(
+			AdditionalText? bannedListFile = options.AdditionalFiles.FirstOrDefault(
 				file => Path.GetFileName( file.Path ) == "BannedTestCategoriesList.txt"
 			);
 
 			if( bannedListFile == null ) {
 				return bannedList.ToImmutableHashSet();
 			}
 
-			SourceText allowedListText = bannedListFile.GetText();
+			SourceText? allowedListText = bannedListFile.GetText();
+
+			if( allowedListText == null ) {
+				throw new Exception( "Couldn't read config" );

```

Comment:
I'm not sure how this would happen so I didn't try hard on this exception; I figure its better than the NullReferenceException we'd otherwise have. The IDE/CLI will tell you which analyzer threw an exception so I figure this is enough hint for now.